finalPrep/README.md

```diff
@@ -18,37 +18,40 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+![](../../../../Desktop/Screen Shot 2021-08-07 at 12.54.52 AM.png)
 - `Add Week 2 Screenshot Here`
+![](../../../../Desktop/Screen Shot 2021-08-07 at 12.55.22 AM.png)
 - `Add Week 3 Screenshot Here`
+![](../../../../Desktop/Screen Shot 2021-08-07 at 12.55.52 AM.png)
```

Comment:
I suspect these worked on your machine but didn't push up to Github as expected. Can you try again? There are a couple different ways to accomplish this and I'm sure you'll be able to google your way to a solution!